Sources/MapboxSpeech/MBSpeechOptions.swift

```diff
@@ -85,3 +85,30 @@ open class SpeechOptions: Codable {
         return params
     }
 }
+
+public extension Locale {
+    
+    /**
+     `String` Returns the identifier of the locale identifier supported by Amazon Polly [`Supported Language`](https://docs.aws.amazon.com/polly/latest/dg/SupportedLanguage.html).
+     
+     While common language identifiers are two-letter `ISO 639-1` standard, three-letter `ISO 639-2` standard or even `RFC 4647` (known as BCP 47), `Amazon Polly` uses `ISO 639-3`
+     W3C language identification which creates incompatibility with `Locale.current`.
+     This computed property either return `Locale.identifier` or the supported version of the unknown code.
+     List of currently unsupported codes :
+     "ar-SA", "zh-CN", "zh-HK", "zh-Hans", "zh-Hant", "zh-TW"
+     */
+    var amazonIdentifier : String {
+        let unsupported : Dictionary<String, String> = [
+            "ar"     : "arb",
+            "zh"     : "cmn-CN"
```

Comment:
We had originally considered fixing the mapping on the server side via mapbox/locale-utils#7 and mapbox/locale-utils#9. Normally, hard-coding language code fallbacks on the client side would be frowned upon, but these particular mappings are unlikely to ever become obsolete as long as the Voice API is based on Amazon Polly. If that ever changes, it would make sense for the server side to create a fallback from these more specific language codes to less specific ones anyways.

This would only be a workaround; other client libraries like mapbox-java would need to make a similar change.

/cc @Guardiola31337